src/Compilers/CSharp/Portable/Errors/SyntaxDiagnosticInfo.cs

```diff
@@ -10,11 +10,6 @@ namespace Microsoft.CodeAnalysis.CSharp
 {
     internal class SyntaxDiagnosticInfo : DiagnosticInfo
```

Comment:
diagnostic-infos are part of diagnostics.  which no longer need to be serialized now.